wallet/chainntfns.go

```diff
@@ -889,6 +890,23 @@ func (w *Wallet) VoteOnOwnedTickets(ctx context.Context, winningTicketHashes []*
 				ticketVoteBits = tvb
 			}
 
+			// When not on mainnet, randomly disapprove blocks based
+			// on the disapprove percent.
+			dp := w.DisapprovePercent()
+			if dp > 0 {
+				if w.chainParams.Net == wire.MainNet {
+					panic("block disapproval percent set on mainnet")
```

Comment:
I wouldn't leave this as a panic, because it could crash voting wallets. I suggest just logging a warning and not doing the disapproving.

Reply:
I think it's just getting the last bit now.